2-mandatory/2-financial-times.js

```diff
@@ -14,7 +22,13 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestTitle = allArticleTitles[1];
```

Comment:
Don't forget that arrays are indexed from 0, not one. If your list would only contain a single article this code would fail

Reply:
right, yeah.. hard to get the new values (0=1) into my brain xD